### ISSUE TITLE
2024, The cheat is NOT working anymore / crashing when injected

### ISSUE THREAD

**Issue by Jopro1990** [alza54/opensource2#23]
Hi, so currently I am using extreme injector manual map. The console says everything is loaded, it doesn't crash when im in the lobby. But as soon as i go into a match, it crashes the entire game.
Could we maybe get an update?

**Comment by sourabhbeni** [comment]
u have to update offsets manually 

PS [check unknowncheats post for this cheat u fill find offsets in comments]

**Comment by Jopro1990** [comment]
> u have to update offsets manually
> 
> PS [check unknowncheats post for this cheat u fill find offsets in comments]

yes if found the offsets, but i don't know where to put these offsets in order for the code to work!

**Comment by Cyk-Fad** [comment]
> u have to update offsets manually
> 
> PS [check unknowncheats post for this cheat u fill find offsets in comments]

could u plz give me a link?

**Comment by DEFS9** [comment]
CGameEntitySystem::GetLocalPlayerController();

**Comment by alza54** [comment]
this ain't a cheat my bro.
